components/webhook-mgt/org.wso2.carbon.identity.webhook.management/src/main/java/org/wso2/carbon/identity/webhook/management/api/model/Webhook.java

```diff
@@ -129,6 +131,9 @@ public List<String> getEventsSubscribed() throws WebhookMgtException {
         if (eventsSubscribed != null) {
             return eventsSubscribed;
         }
+        if (StringUtils.isEmpty(getUuid())) {
```

Comment:
It should not be getUUID. This should be getID. we may internally have a UUID which doesn't matter at contract level

Reply:
removed it. https://github.com/wso2/carbon-identity-framework/pull/6821/commits/e674d7bb37b7c289d2084dfa13087e19dc16bd5c